src/main/webapp/app/entities/student-with-team.model.ts

```diff
@@ -0,0 +1,6 @@
+export class StudentWithTeam {
+    public Name: string;
```

Comment:
Properties need to be in [camelCase](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client/#names)

Reply:
That is intentional, since the requested JSON on issue has that notation, it automatically parses it. And throws error if something is missing.